aiida/cmdline/commands/cmd_computer.py

```diff
@@ -403,17 +403,18 @@ def computer_list(all_entries, raw):
 
     if not raw:
         echo.echo_info('List of configured computers')
+        echo.echo_info('Computers that are configured and ready to use are marked with an asterisk')
         echo.echo_info("Use 'verdi computer show COMPUTERNAME' to display more detailed information")
 
     computers = Computer.objects.all()
     user = User.objects.get_default()
 
-    if not computers:
+    if not computers and not raw:
         echo.echo_info("No computers configured yet. Use 'verdi computer setup'")
 
     sort = lambda computer: computer.name
     highlight = lambda comp: comp.is_user_configured(user) and comp.is_user_enabled(user)
-    hide = lambda comp: not (comp.is_user_configured(user) and comp.is_user_enabled(user)) and not all_entries
+    hide = lambda comp: comp.is_user_configured(user) and not comp.is_user_enabled(user) and not all_entries
```

Comment:
The confusing part here maybe that `is_user_configured` means there is a `AuthInfo` for the given `User` and `Computer`, whereas `is_user_enabled` means whether said `AuthInfo` has it's enabled property set to `True`. This can be set to `False` to stop calculations being submitted with that `AuthInfo` (even though this is currently not respected by the engine, for which there is an issue). None of this logic deals with the computers or authinfos of _other_ users. Note also that we do not even have the option to show the computers of any given specific user. It just always takes the current default user. The concept of multiple users is not really well worked out yet, which is why all these commands that deal with stuff from other users are confusing, I feel